src/eventListeners/index.ts

```diff
@@ -0,0 +1,64 @@
+import { utils } from 'ethers';
+
+import { verbose } from '~utils';
+import { ContractEventsSignatures } from '~types';
+
+import { EventListener } from './types';
+
+export * from './colony';
+export * from './network';
+export * from './extensions';
+export * from './token';
+
+let listeners: EventListener[] = [];
```

Comment:
I think this would be possible (and as you point out beneficial) in the version you reviewed, now changing this would break`removeVotingReputationListeners` and `getMatchingListener`. Let's come back to it if it proves to be a bottleneck